.claude/agents/test-runner.md

```
@@ -0,0 +1,123 @@
+
+# 特定ファイルのみ実行する場合
+npm test -- --testPathPattern="[パターン]" --verbose 2>&1
+```
+
+## メインエージェントへの報告形式
+
+テスト完了後、以下の形式でメインエージェントに報告してください：
+
+### 全テスト成功の場合
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)